docs/source/sdk_chaincode.md

```diff
@@ -0,0 +1,53 @@
+Install Fabric Application SDKs
+-----------------------------------
+
+**PS:** For Network Administrators interested in running and managing a Fabric Network only, you may skip installing the following SDKs and APIs.
```

Comment:
Instead of **PS**, we've used an **Audience** header instead.
See https://hyperledger-fabric.readthedocs.io/en/latest/tutorial/commercial_paper.html 
https://hyperledger-fabric.readthedocs.io/en/latest/tutorial/commercial_paper.html for an example of that pattern which I think could work nicely here.

Reply:
Updated.